dffml/secret/file.py

```diff
@@ -0,0 +1,46 @@
+import os
+import hashlib
+import asyncio
+import pathlib
+from typing import Union,Any
+
+from ..base import config
+from ..util.entrypoint import entrypoint
+from .base import BaseSecretContext, BaseSecret
+
+from dffml.high_level import Record
+from ..source.memory import MemorySourceContext
+from ..source.ini import INISourceConfig,INISource
+
+
+
+class FileSecretContext(BaseSecretContext, MemorySourceContext):
+    def __init__(self,*args,**kwargs):
+        super().__init__(*args,**kwargs)
+
+
+    async def get(self, name: str) -> Union[None, str]:
+        with open(self.parent.config.filename,'r+') as fd:
```

Comment:
It looks like we shouldn't need this, because `FileSource` calls `load_fd` of `INISource` on `__aenter__`, which will load the data into `self.mem`